cime_config/machines/scripts/cmake3ify-package

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env python3
```

Comment:
Is this script really worth committing to the repo? I assume after this big effort, it will likely never need to be used again...

Reply:
I don't care too much one way or the other. This script will potentially be useful in the weeks ahead as we continue on with this effort. This directory is where i've been putting miscellaneous build-related scripts.